Makefile

```diff
@@ -180,20 +180,24 @@ SMOKEFLAGS?=--color --diff=git
 
 .PHONY: check-only
 check-only:
-	@${MAKE} build
-	@${MAKE} install
-	@${MAKE} test
-	@${MAKE} smoke
-	@${MAKE} check-format-juvix-examples
-	@${MAKE} typecheck-juvix-examples
-	@${MAKE} check-ormolu
-	@export SKIP=ormolu,format-juvix-examples,typecheck-juvix-examples
-	@${MAKE} pre-commit
+	@${MAKE} build \
+		&& ${MAKE} install \
+		&& ${MAKE} test \
+		&& ${MAKE} smoke \
+		&& ${MAKE} check-format-juvix-examples \
+		&& ${MAKE} typecheck-juvix-examples \
+		&& ${MAKE} check-ormolu \
+		&& export SKIP=ormolu,format-juvix-examples,typecheck-juvix-examples \
+		&& ${MAKE} pre-commit
 
 .PHONY: check
 check: clean
 	@${MAKE} check-only
 
+.PHONY : bench
+bench: runtime submodules
+	@stack bench
```

Comment:
Do we want to append one of the stack flags variables to this command?

Reply:
We should yes, also use ${STACK} similar to the use for make.